justfile

```
@@ -174,6 +174,10 @@ dj args="":
 stats media="images":
     curl "http://localhost:8000/v1/{{ media }}/stats/"
 
+# Attach to ipython
+ipython:
+    docker-compose exec web ipython
```

Instead of pure `ipython`, I believe `python manage.py shell` would be more useful as it sets up some Django magic automatically.
```suggestion
    docker-compose exec web python manage.py shell
```